From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Wed, 16 Aug 2023 18:36:38 +0200
Subject: build: Bump meson requirement

We need at least 0.60 to pass multiple arguments to `dependency()`.
That release dates back to 2021, so should be uncontroversial.

Origin: https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/3187
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 3589c39..5af3446 100644
--- a/meson.build
+++ b/meson.build
@@ -1,6 +1,6 @@
 project('mutter', 'c',
   version: '45.beta.1',
-  meson_version: '>= 0.58.0',
+  meson_version: '>= 0.60.0',
   license: 'GPLv2+'
 )
 
